src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { Fragment } from 'react';
+
+import { storiesOf } from '@storybook/react';
+
+import { COLORS } from '../../colors';
+
+import { Button } from './Button';
+
+storiesOf('Button', module)
+	.addParameters({ jest: ['Button'] })
+	.add('Buttons', () => (
+		<Fragment>
+			<div className="u-spacer-bottom">
```

Comment:
opmerking voor mono: text in button is niet gecentreerd:
![image](https://user-images.githubusercontent.com/1710840/58540734-46bb5500-81fa-11e9-9ac8-d9a99b4b11b3.png)